.github/ISSUE_TEMPLATE/bug_report_form.yml

```
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report
+title: ""
+labels: [bug, triage]
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Before you create the issue:
+        * Search for similar report among other reported issues.
+        * Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
+#  - type: input
+#    id: contact
+#    attributes:
+#      label: Contact Details
+#      description: How can we get in touch with you if we need more info?
+#      placeholder: ex. email@example.com
+#    validations:
+#      required: false
+- type: dropdown
+id: Mautic Version
+attributes:
+label: Mautic Version
+description: What version of Mautic you are using? Please test to reproduce your bug with latest stable version of Mautic which might contain new fixes.
+options:
+  - 4.0.x series
+  - 3.3.3
+  - 3.3.x series
+  - 3.2.x series (not supported)
+  - 3.1.x series (not supported)
+  - 2.x series (not supported)
+validations:
+required: true
+- type: input
+   id: PhP version
+   attributes:
+     label: PhP version
+     description: What PHP version are you using in your environment?
+     placeholder: ex. 7.3.27
+   validations:
+     required: true
+- type: dropdown
+  id: browsers
+  attributes:
+    label: What browsers are you seeing the problem on?
+    multiple: true
+    options:
+      - Firefox
+      - Chrome
+      - Safari
+      - Microsoft Edge
+      - Not relevant
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      value: "A bug happened!"
+    validations:
+      required: true
+   - type: textarea
+     id: what-happened
+     attributes:
+       label: How can we reproduce this issue?
+       description: Explain us carefuly how we can reproduce the bug you faced.
+       value: "1.
+       2."
+     validations:
+       required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  # - type: checkboxes
```

```suggestion
   - type: checkboxes
```
I would like to leave a checkbox to state that they have read and agree to abide with the Code of Conduct (please uncomment this whole section :)